frame/vamm/src/lib.rs

```diff
@@ -1231,15 +1221,10 @@ pub mod pallet {
 		) -> Result<DecimalOf<T>, DispatchError> {
 			let now = Self::now(now);
 			let weight_now: MomentOf<T> =
-				std::cmp::max(1_u64.into(), now.saturating_sub(last_twap_timestamp));
+				cmp::max(1_u64.into(), now.saturating_sub(last_twap_timestamp));
```

Comment:
Now that I think about it, does `Ord` automatically come with `.max(...)` for types that implement it?

Reply:
Oh, good one!